keyboards/wuque/mammoth75x/mammoth75x.c

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2021 wuquestudio
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "mammoth75x.h"
+
+
+static uint8_t encoder_state[ENCODERS] = {0};
```

Comment:
Ideally, this code should only be used if via is enabled. 
```suggestion
#ifdef VIA_ENABLE
static uint8_t encoder_state[ENCODERS] = {0};
```